### PR TITLE
Concourse setup

### PR DESCRIPTION
**Pull request by fshuva**
Concourse Setup to run a pipeline of IBM HA offering. Will perform Terraform plan, apply and destruction

### CODE REVIEW

**Review by Joel-Cripps — APPROVED** [review]
looks good, lets add a description to the PR

File: ci/concourse/terraform-dockerimage-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,191 @@
+resources:
+- name: terraform-ibm-ha-repo
+  type: git
+  icon: github
+  source:
+    uri: https://github.com/fshuva/ibm-fortigate-AP-HA-terraform-deploy.git
+    branch: concourse_setup
+- name: terraform-0-13-7
+  type: docker-image
```
Comment (jamie-pate) [inline ci/concourse/terraform-dockerimage-pipeline.yml:9]: should prefer the newer resource type 'registry-image'
See https://github.com/concourse/registry-image-resource#comparison-to-docker-image-resource

File: ci/concourse/terraform-dockerimage-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      inputs:
+      - name: terraform-ibm-ha-repo
+      outputs:
+      - name: terraform-ibm-ha-repo
+      platform: linux
+      run:
+        path: terraform
+        dir: terraform-ibm-ha-repo
+        args:
+          - destroy
```
Comment (jamie-pate) [inline ci/concourse/terraform-dockerimage-pipeline.yml:152]: Should probably run prettier to be sure there's one style of indentation used here..

(prettier will prefer the indented array values like `args:` uses)

**Review by jamie-pate — CHANGES_REQUESTED** [review]

File: ci/concourse/terraform-dockerimage-pipeline.yml (hunk 1 above)
Comment (fshuva) [reply ci/concourse/terraform-dockerimage-pipeline.yml]: Done

File: ci/concourse/terraform-dockerimage-pipeline.yml (hunk 2 above)
Comment (fshuva) [reply ci/concourse/terraform-dockerimage-pipeline.yml]: TIL about prettier. I will use that before pushing commits now.

**Review by Joel-Cripps — APPROVED** [review]

File: ci/concourse/terraform-dockerimage-pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,191 @@
+resources:
+    - name: terraform-ibm-ha-repo
+      type: git
+      icon: github
+      source:
+          uri: https://github.com/fshuva/ibm-fortigate-AP-HA-terraform-deploy.git
+          branch: concourse_setup
+    - name: terraform-0-13-7
```
Comment (JaydenLiang) [inline ci/concourse/terraform-dockerimage-pipeline.yml:8]: don't think you need to construct the name with the version unless you use more than 1 version in a single pipeline.
can simply name it terraform-tool or terraform-image

**Review by JaydenLiang — CHANGES_REQUESTED** [review]
please see my previous comment

**Review by jamie-pate — APPROVED** [review]

**Pull request merged**